tests/metrics/test_base.py

```diff
@@ -0,0 +1,31 @@
+import re
+
+from mlflow.metrics.base import EvaluationExample
+
+
+def test_evaluation_example_str():
+    example1 = str(
+        EvaluationExample(
+            input="This is an input",
+            output="This is an output",
+            score=5,
+            justification="This is a justification",
+            variables={"foo": "bar"},
+        )
+    )
+    example1_expected = """
+        Input: This is an input
+        Provided output: This is an output
+        Provided foo: bar
+        Score: 5
+        Justification: This is a justification
+        """
+    assert re.sub(r"\s+", "", example1_expected) == re.sub(r"\s+", "", example1)
```

Comment:
do we care about the formatting of these strings? if so, should we use `strip()` or maybe not even remove whitespace during these checks.

Reply:
I don't think we care about the exact spacing so this test should ideally be ok if the contents are generated correctly.